docs/src/proposals/scrambled-tx.md

```diff
@@ -0,0 +1,335 @@
+---
+title: Scrambled Transaction to combat against MEV
+---
+
+### Program Statement
+
+The existence of MEV isn't ideal and costly.
+
+MEV motivates transaction reordering, censoring and copying. Also, it causes
+cluster destabilization because of incentivised forking.
+
+The root problem is that there is no rule in the pursuit of MEV frontier, except
+economic rationality. While this anarchism is welcomed, stemmed from the
+blockchain philosophy, it's also desirable for such an activity to economically
+impractical in the first place.
+
+Any temporal market imbalance should be corrected strictly on the grounds of a
+fair game, basically the golden first-come-first-served rule (with optional
+bidding for priority).
+
+Note that this proposal doesn't fully mitigate one of MEV types, unlike all
+other fully-neutralized types: back-running. Back-running can be exercised
+while the golden rule is strictly and fairly enforced.
+
+### Summary of Proposed Solution
+
+This proposal tries to combat against MEV, at the cost of (slight) global
+induced latency of cluster as a whole.
+
+At highest level, a full mitigation of MEV can be attained _if leaders (and
+searchers) can't know the contents of other's transactions until finishing to
+packing and ordering them into a new block_. In addition to this, the
+transaction details must also be deterministically revealed after the ordering
+of transactions is committed.
+
+So, naive 2-step transaction submission (encrypted tx, then its key) won't work,
+let alone increased latency.
+
+Instead, this proposal uses a special transaction construction _only to delay
+the revelation of its content for the short amount of time tied to confirmation
+by the cluster_. Specifically, transactions are encrypted by a private key
+derived from the recursive sha256, and its seed and cipher text are submitted to
+the cluster at the same time.
+
+This construction is called *scrambling* across this proposal.
+
+Conversely, the forced computation to reveal the transaction details by solving
+the recursive sha256 is called *descrambling*.
+
```

Comment:
document about `underlying ideological/ecnomical motivation and intended ecosystem evolution.`.

Reply:
ref: https://discord.com/channels/938287290806042626/938287767446753400/958022933551415316


thanks for the suggestion. i studied about pbs. i think my proposal is similar to pbs. it seems my proposal can be rephrased as hypothetical separation of proposer and builder via forced temporal confidentiality of transactions, thanks to being scrambled only  at the time of building. Still, i don't think my proposal necessarily equates to being negative externalities from MEV activities:
1. solana's bidding will be similar to localized first-price sealed-bid auction. so it shouldn't badly affect the overall performance. current mainnet-beta state of affair is just resulting from bad impl and tragedy of commons by bots.
2. it seems that pbs introduces fair amount of complexity and variable latency than this proposal. this in turn creates another systemic risk. so, I'm saying monolithic system is preferred, assuming there should be little negative externalities.
3. first of all, the existence of arbitrary reordering ability is the root problem for blockchain. and pbs is trying to cover up by introducing another indirection. I'm trying to plug it altogether instead. this is like paradigm shift of jquery => reactjs. imo, this reordering is even inferior characteristics than centralized systems. time (fcfs) and random (uniform expectation) should be strived with strong emphasis for blockchain fairness blurb. 
4. I don't think validator's rewards should be that lucrative for overall ecosystem sustainability. operating it should rather be stable business activity. they should be directed to focus on highest possible uptime and gain rather stable rewards from doing so without tapping into mev revenue source.
5. in other words, validator's mev income is kind of middleman. so, ultimately, i think searchers should be able pay back to target tx submitters, not miners. externalizing the mev marketplace from on-chain just hinders that. so, i'm saying mev equivalent for jupiter could be a possibility. thanks to solana's composability and no sharding.